src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import static duke.utils.Ui.logo;
+import static duke.utils.Ui.wrapWithLines;
+
+import java.io.File;
+import java.util.Scanner;
+
+import duke.commands.CommandHandler;
+import duke.commands.CommandHandlerFactory;
+import duke.data.Storage;
+import duke.exceptions.DukeException;
+import duke.tasks.TaskList;
+
+
+/**
+ * Duke is a program that helps its user to store tasks.
+ */
+public class Duke {
+
+    private void initialise() {
+        wrapWithLines("Hello from\n" + logo);
+
+        File storageDirectory = new File("./data");
+        if (!storageDirectory.exists()) {
+            if (!storageDirectory.mkdir()) {
```

Comment:
Perhaps you could use this method to make your if else statement not nested for better readability :)
```suggestion
            if (!storageDirectory.mkdir() && !storageDirectory.mkdir()) {
```

Reply:
Thank you for your suggestion!